### PR TITLE
FUSETOOLS2-681 - provide test for default task

### PR DESCRIPTION
**Pull request by apupier**
requires https://github.com/camel-tooling/vscode-camelk/pull/569 and https://github.com/camel-tooling/vscode-camelk/pull/570

### CODE REVIEW

**Comment by apupier** [comment]
again reporting duplicated code on untouched code in this PR

File: src/task/CamelKTaskDefinition.ts [hunk 1]
```diff
@@ -20,6 +20,8 @@ import * as kamel from '../kamel';
 import * as vscode from 'vscode';
 import * as IntegrationUtils from '../IntegrationUtils';
 
+export const TASK_NAME_DEFAULT_DEPLOY_FROM_ACTIVE_EDITOR = "Start in dev mode Camel K integration opened in active editor";
```
Comment (lhein) [inline src/task/CamelKTaskDefinition.ts:23]: I find that label a bit strange. Wouldn't it be better to name it like:
`Start currently opened Camel K integration in dev mode` ?

File: src/task/CamelKTaskDefinition.ts [hunk 2]
```diff
@@ -58,7 +60,7 @@ export class CamelKTaskProvider implements vscode.TaskProvider {
 		this.tasks = [];
 		let taskDefinition = {
 			"type": CamelKTaskProvider.START_CAMELK_TYPE,
-			"label": "Start in dev mode Camel K integration opened in active editor",
+			"label": TASK_NAME_DEFAULT_DEPLOY_FROM_ACTIVE_EDITOR,
```
Comment (lhein) [inline src/task/CamelKTaskDefinition.ts:63]: The name of the constant does not really fit what it does when reading the label of that constant. Maybe we can improve that a bit?

File: src/task/CamelKTaskDefinition.ts (hunk 1 above)
Comment (apupier) [reply src/task/CamelKTaskDefinition.ts:23]: maybe. This has not been modified in this PR. Creating an issue to discuss to provide a better naming https://issues.redhat.com/browse/FUSETOOLS2-785

**Review by lhein — APPROVED** [review]

**Pull request merged**